src/Unity.Mathematics/Geometry/MinMaxAABB.cs

```diff
@@ -1,49 +1,332 @@
 using System;
+using System.Runtime.CompilerServices;
 using static Unity.Mathematics.math;
 
 namespace Unity.Mathematics.Geometry
 {
+    /// <summary>
+    /// Axis aligned bounding box (AABB) stored in min/max form.
+    /// </summary>
+    /// <remarks>
+    /// Axis aligned bounding boxes (AABB) are boxes where each side is parallel with one of the Cartesian coordinate axes
+    /// X, Y, and Z. AABBs are useful for approximating the region an object (or collection of objects) occupy and quickly
+    /// testing whether or not that object (or collection of objects) is relevant. Because they are axis aligned, they
+    /// are very cheap to construct and perform overlap tests with them.
+    /// </remarks>
     [System.Serializable]
     internal struct MinMaxAABB : IEquatable<MinMaxAABB>
     {
+        /// <summary>
+        /// The minimum point contained by the AABB.
+        /// </summary>
+        /// <remarks>
+        /// If any component of <see cref="Min"/> is greater than <see cref="Max"/> then this AABB is invalid.
+        /// </remarks>
+        /// <seealso cref="IsValid"/>
         public float3 Min;
+
+        /// <summary>
+        /// The maximum point contained by the AABB.
+        /// </summary>
+        /// <remarks>
+        /// If any component of <see cref="Max"/> is less than <see cref="Min"/> then this AABB is invalid.
+        /// </remarks>
+        /// <seealso cref="IsValid"/>
         public float3 Max;
 
-        public bool IsEmpty
+        /// <summary>
+        /// Constructs the AABB with the given minimum and maximum.
+        /// </summary>
+        /// <remarks>
+        /// If you have a center and extents, you can call <see cref="CreateFromCenterAndExtents"/> or <see cref="CreateFromCenterAndHalfExtents"/>
+        /// to create the AABB.
+        /// </remarks>
+        /// <param name="min">Minimum point inside AABB.</param>
+        /// <param name="max">Maximum point inside AABB.</param>
+        public MinMaxAABB(float3 min, float3 max)
```

Comment:
Should this also have `[MethodImpl(MethodImplOptions.AggressiveInlining)]`? According to MSDN, `MethodImpl` can also target a ctor.